### PR TITLE
FEATURE: SETI-2159 - Use mvn release plugin

### PR DESCRIPTION
**Pull request by pgressa**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -36,34 +36,22 @@
   </properties>
 
   <scm>
-    <connection>scm:git:ssh://github.com/zeroturnaround/build-flow-test-aggregator.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/zeroturnaround/build-flow-test-aggregator.git</developerConnection>
-    <url>https://github.com/zeroturnaround/build-flow-test-aggregator</url>
-    <tag>build-flow-test-aggregator-1.2</tag>
+    <connection>scm:git:git@github.com:gooddata/build-flow-test-aggregator.git</connection>
```
Comment (michalvanco) [inline pom.xml]: check the branch pls

**Comment by michalvanco** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/build-flow-test-aggregator-opensource-repository-check/1/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/java-libraries/job/build-flow-test-aggregator-tests-zuul-el7/1/ : SUCCESS in 2m 58s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/build-flow-test-aggregator-merge-compliance-check/1/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/java-libraries/job/build-flow-test-aggregator-tests-zuul-el7/2/ : SUCCESS in 1m 40s

**Pull request merged**